### ISSUE TITLE
V4: Some Accessibility issues

### ISSUE THREAD

**Issue by christopherpross** [prologueapp/Prologue#65]
Hey,
thank you for your awesome work. I am testing your new v4 prologue app now for a few days.
I am blind and using VoiceOver on Ios to interact with apps. 
At this moment, many buttons in the App are not labeled for Screen readers. The books are not read out, I have to click on a random button to get the title. Also the button to go to the downloaded books and the server list is not labeled. 
I know that you had already spend time in the v3 Prologue in the accessibility, I hope you will fix this issues regarding accessibility also in v4. Of course it is a alpha, this issue should be only a little reminder. :-) 

Thank you and when you need feedback, I am here.

all the best,
Christopher

**Comment by lcharlick** [comment]
I just did another accessibility pass, including adding back the rotor actions from v3. Let me know here or open another issue if I've missed anything 🙏
